crates/concrete_check/src/linearity_check.rs

```diff
@@ -0,0 +1,1086 @@
+use std::collections::HashMap;
+
+use concrete_session::Session;
+
+use self::errors::LinearityError;
+pub mod errors;
+
+use std::path::PathBuf;
+//use concrete_ir::{ProgramBody, FnBody};
```

Comment:
please remove these commented imports. They can be added afterwards if they are needed.

Reply:
Done